Signal/src/ViewControllers/SignalsViewController.m

```diff
@@ -603,7 +603,6 @@ - (NSNumber *)updateInboxCountLabel {
     [_segmentedControl setTitle:unreadString forSegmentAtIndex:0];
     [_segmentedControl.superview setNeedsLayout];
     [_segmentedControl reloadInputViews];
-    [[UIApplication sharedApplication] setApplicationIconBadgeNumber:badgeNumber.integerValue];
```

Comment:
Since messagesManager is *always* observing and keeping applicationIconBadgeNumber up to date, we don't need to do any such thing here.